code/modules/admin/sql_notes.dm

```diff
@@ -0,0 +1,186 @@
+/proc/add_note(target_ckey, notetext, timestamp, adminckey, logged = 1, server)
+	if(!check_rights(R_INVESTIGATE, 1, usr)) return
+
+	if(!dbcon.IsConnected())
+		to_chat(usr, "<span class='danger'>Failed to establish database connection.</span>")
+		return
+
+	if(!target_ckey)
+		var/new_ckey = ckey(input(usr,"Who would you like to add a note for?","Enter a ckey",null) as text)
+		if(!new_ckey)
+			return
+		new_ckey = sanitizeSQL(new_ckey)
+		var/DBQuery/query_find_ckey = dbcon.NewQuery("SELECT ckey FROM erro_player WHERE ckey = '[new_ckey]'")
```

Comment:
Needs backticks for columns and tables
dbcon.NewQuery("SELECT \`ckey\` FROM \`erro_player\` WHERE \`ckey\` = '[new_ckey]'")